config/src/config/storage_config.rs

```diff
@@ -65,11 +64,6 @@ pub struct StoragePrunerConfig {
     /// being big in size, we might want to configure a smaller window for state store vs other
     /// store.
     pub ledger_store_prune_window: Option<u64>,
-
-    /// Maximum version to prune per batch, should not be too large to avoid spike in disk IO caused
-    /// by large batches in the pruner.
-    pub max_version_to_prune_per_batch: Option<u64>,
```

Comment:
hmm, when we shrink the prune_window, will it end up in a super large batch?

Reply:
Good catch, Added the batch pruning functionality back as discussed offline.